src/Data/Parity/HomomorphismNatParity.agda

```diff
@@ -0,0 +1,103 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some properties about parities
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Parity.HomomorphismNatParity where
```

Comment:
While I agree we should do something about https://github.com/agda/agda-stdlib/issues/1871 in the medium to long term, I think it would be better to stick with the current convention used everywhere else in the library and put this in `Data.Parity.Properties`, rather than have two competing conventions.

Reply:
Done. This module also now removed from the branch.